src/main/java/nextmainfocus/jwt/JwtService.java

```diff
@@ -0,0 +1,115 @@
+package nextmainfocus.jwt;
+
+import java.security.Key;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Function;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.stereotype.Service;
+
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.ExpiredJwtException;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.MalformedJwtException;
+import io.jsonwebtoken.SignatureAlgorithm;
+import io.jsonwebtoken.UnsupportedJwtException;
+import io.jsonwebtoken.io.Decoders;
+import io.jsonwebtoken.security.Keys;
+import io.jsonwebtoken.security.SignatureException;
+
+@Service
+// @ConfigurationProperties("jwt")
+public class JwtService {
+	private static final Logger logger = LoggerFactory.getLogger(JwtService.class);
+
+	private long expiration = 18000000; // 5 hours in milliseconds 5 * 60 * 60 * 1000
```

Comment:
good practice to add _ms to the var'a name

Reply:
Why exactly? ;) Adding any meta-information (typings, allocation, goal, etc.) was a part of [Hungarian notation](https://en.wikipedia.org/wiki/Hungarian_notation) and is not considered a best practice in 2023. Btw, CAPS for const in Java is a part of that, currently is an inevitable legacy. 🤗
Btw, this [magic number](https://en.wikipedia.org/wiki/Magic_number_(programming)) should be moved into an external config file (according to #25)